operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -76,6 +76,14 @@ spec:
 #    ---
 #    strategy: ""
 
+##########
+# Determines whether to show links of Grafana and Traces or not depending on user
+# access to Istio control plane pods. Kiali won't show Grafana and Tracing integration
+# in case the user doesn't have permissions to list pods of IstioNamespace.
+# When empty, its default will be `true`.
+#    ---
+#  authorize_external_services: false
```

Comment:
OK, after looking at this closely, this is wrong. All commented out settings here have values set to their defaults. I think you want this to say "true". This .yaml file is used for documentation - people go here to see the settings, descriptions, and the default values (read the top comment in this file, it explains this). So change this to:
```suggestion
#  authorize_external_services: true
```